packages/caver-klay/caver-klay-kct/kctHelper.js

```diff
@@ -20,6 +20,11 @@ const _ = require('lodash')
 const BigNumber = require('bignumber.js')
 const { isBigNumber } = require('../../caver-utils')
 
+const KCT_TYPE = {
```

Comment:
Actually, this categorization is not good to me. What if we have several fungible token standard?
I think if we separate the validation function, we don't need this categorization.

Reply:
Okay. I will separate the functions!